.github/actions/sam-build/action.yaml

```diff
@@ -0,0 +1,24 @@
+name: sam-build
```

Comment:
I'm wondering why you decided to put this in a `sam-build` sub directory?  why not just put the file under `actions` dir?

Reply:
is there any motivation to figure this out?

Reply:
blarg, it tries to treat `sam-build.yaml` like a directory instead of a file: https://github.com/Sage-Bionetworks-IT/lambda-template/actions/runs/6179349694/job/16774149759?pr=26

Reply:
ok, good to know and thanks for investigating.